ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+puts "How many employees would you like to process?"
+number_of_employees=gets.chomp.to_i
+for i in 1..number_of_employees
+
+	puts "What is your name?"
+	your_name=gets.chomp
+	puts "How old are you?"
+	your_age=gets.chomp.to_i 
+	puts "What year were you born?"
+	year_born=gets.chomp.to_i
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+	garlic_bread=gets.chomp
+	case garlic_bread
+	when "Yes","yes","Y","y","Yeah","yeah"
+		garlic_bread = true
+	else garlic_bread = false
+	end
+
+	puts "Would you like to enroll in the company's health insurance?"
+	health_insurance=gets.chomp
+
+	case health_insurance
+	when "Yes","yes","Y","y","Yeah","yeah"
+		health_insurance = true
+	else health_insurance = false
+	end
+
+	time = Time.new

```

Comment:
Since you are only referencing the year, it makes sense to make `time = Time.new.year`